src/intTest/java/com/box/sdk/EventStreamIT.java

```diff
@@ -6,8 +6,10 @@
 
 import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.TimeUnit;
+import org.junit.Ignore;
 import org.junit.Test;
 
+@Ignore
```

Comment:
I suppose that there is a problem with this test, right?

Reply:
I forgot to enable that.